src/BufferCopy/ColorBufferToRDRAM_GLES.cpp

```diff
@@ -62,17 +62,16 @@ void ColorBufferToRDRAM_GLES::_initBuffers(void)
 	m_window->reallocate(m_pTexture->realWidth, m_pTexture->realHeight,
 		PIXEL_FORMAT_RGBA_8888, GraphicBuffer::USAGE_SW_READ_OFTEN | GraphicBuffer::USAGE_HW_TEXTURE);
 	EGLint eglImgAttrs[] = { EGL_IMAGE_PRESERVED_KHR, EGL_TRUE, EGL_NONE, EGL_NONE };
-	m_image = eglCreateImageKHR(eglGetDisplay(EGL_DEFAULT_DISPLAY), EGL_NO_CONTEXT,
-		EGL_NATIVE_BUFFER_ANDROID, (EGLClientBuffer)m_window->getNativeBuffer(), eglImgAttrs);
+
+	if(m_image == 0)
+	{
+		m_image = eglCreateImageKHR(eglGetDisplay(EGL_DEFAULT_DISPLAY), EGL_NO_CONTEXT,
+			EGL_NATIVE_BUFFER_ANDROID, (EGLClientBuffer)m_window->getNativeBuffer(), eglImgAttrs);
+	}
 }
 
 void ColorBufferToRDRAM_GLES::_destroyBuffers(void)
 {
-	if(m_image != 0)
-	{
-	    eglDestroyImageKHR(eglGetDisplay(EGL_DEFAULT_DISPLAY), m_image);
-	    m_image = 0;
-	}
 }
 
```

Comment:
This makes _destroyBuffers method useless. Please remove it from ColorBufferToRDRAM and all its successors.